### ISSUE TITLE
[BUG] Accessibility Issues report

### ISSUE THREAD

**Issue by remmycat** [deven-org/deven-website#130]
There are a few accessibility issues I, and netlify-plugin-a11y found on the website.
These likely need to be sorted into workable tasks, this ticket is only used for the initial report of the findings.

Manual findings on macOS 13.5.2 with Safari 16.6, using the VoiceOver screen reader.
I have only manually tested the main page so far, there might be further issues on the others.

**edit: i changed this to a task list for me to work on for now, leftovers will get a proper issue describing the problem**
**edit: i checked off all items that I believe to have fixed in PR #131**

- [x] Submenu
  - [x] The "Discover" toggle is not usable using keyboard navigation. (not reachable/usable via tab key and co)
  - [x] The "Discover" toggle is only announced as "clickable" to VoiceOver. It should be a real `<button>`, and additionally have an `aria-expanded` state to signal if the submenu is open or not.
  - [x] In the open submenu, the close button is also not a real button and not reachable with either screenreader or standard keyboard navigation
  - [x] The close button has no label
  - [x] After opening one of the links in the discover menu (they are `#links` leading to elements on the same page), the screenreader's "cursor" is not moving to that section, making the links a bit useless. From some research, this might be an issue with VoiceOver, that has no easy fix from our side. Edit: Turns out this was caused by the submenu closing immediately after clicking a link, which bugs VoiceOver. I deactivated this for now, leaving the menu open until manually closed.
  - [x] Automatic testing noticed that the `"subnav"` ID is used twice, that is invalid HTML
  - [x] Contributors menu item is not clickable while submenu is open
- [ ] #132
  - [ ] These need design changes
  - [ ] The small logo in the "Hello" text could easily be changed to the black variant that is shown in the menu
- [x] "Product Engineering" and "Standardization" elements under the logo are`<button>`s without any use / handler. They should either have a use, or not be buttons.
- [x] The link with the down arrow is not labelled (for screen readers)
- [x] Emojis in the texts need to be hidden for screen readers and labelled by the word we mean, to make sense when read out.
  - Excerpt from VoiceOver: "We know it's not always a walk in the Leafy Tree, though. Sometimes you may Person Running into obstacles..." 😅 
- [x] "Tools" links
  - [x] Currently these are `<a>` tags inside `<button>` tags, that's invalid HTML (they should only be `a` tags, the buttons aren't doing anything)
  - [x] Currently the buttons/links aren't labelled by their text
  - [x] Nitpick: The links would likely be easier to use if they wrapped around the whole Text & arrow, instead of being only on the arrow. This would also solve the labelling.
  - [x] `slidePrev` and `slideNext` IDs are used multiple times (invalid HTML)
- [x] Current Team
  - [x] Headline "Current Team" should be an `h2`
  - [x] Author avatars have a useless `alt` text, as it is the same text as underneath, when using a screen reader this just means the text will be read twice. The alt text should either be empty, or include a visual description of the person depicted. (This is a decision that needs to be made on the basis of it being contextually relevant or not what the team members look like). Edit: I set the alt to "" for now, we could still add visual descriptions in the future.
- [x] Nitpick: "Contributors" currently has a unicode arrow, as part of its label, that is read out, even though it is used decoratively and should be hidden to a screen reader.
- [x] Latest news
  - [x] The section should have a heading or at least a hidden accessible label that is announced to screen readers.
  - [x] the post images and author avatars have the same issue as team avatars - the `alt` text should either be empty (`alt=""`) or, if relevant in their context, provide a description of the image.
- [x] ~~Nitpick: It would be easier to navigate the page using a screen reader by making the clearly seperated sections of the page into `<section>` tags with an accessible label (e.g. `aria-label`), which will make them region landmarks.~~ Edit: I tried this and I think on this short website it introduced more duplications and weird screenreader-glitches than actual benefits over the menu or headline-based navigation

<summary>
Unfold to see accessibility report by netlify-plugin-a11y (should all be covered by the mentioned points)
<details>

```

[404 - Not Found (docs/404.html)] error 1: Duplicate id attribute value subnav found on the web page. (<div class=subnav-content id=subnav>
                  <ul>
       ...</div>)
[DEVENorg (docs/index.html)] error 1: Duplicate id attribute value subnav found on the web page. (<div class=subnav-content id=subnav>
                  <ul>
       ...</div>)
[DEVENorg (docs/index.html)] error 1: Anchor element found with a valid href attribute, but no link content has been supplied. (<a class=link-button href=#about-deven>
                        <span ...</a>)
[DEVENorg (docs/index.html)] error 1: This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined. (<button>
                <a href=./doc...</button>)
[DEVENorg (docs/index.html)] error 1: Anchor element found with a valid href attribute, but no link content has been supplied. (<a href=./documentation-skeleton>
                    <svg width...</a>)
[DEVENorg (docs/index.html)] error 1: This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined. (<button>
                <a href=./tel...</button>)
[DEVENorg (docs/index.html)] error 1: Anchor element found with a valid href attribute, but no link content has been supplied. (<a href=./telemetry>
                    <svg width...</a>)
[DEVENorg (docs/index.html)] error 1: Duplicate id attribute value slidePrev found on the web page. (<svg width=18 height=18 viewBox=0 0 36 26 fill=none xmlns=http://www.w3.org/2000/svg id=slidePrev>
                    <path d=M...</svg>)
[DEVENorg (docs/index.html)] error 1: Duplicate id attribute value slideNext found on the web page. (<svg width=18 height=18 viewBox=0 0 36 26 fill=none xmlns=http://www.w3.org/2000/svg id=slideNext>
                    <path d=M...</svg>)
[DEVENorg (docs/index.html)] error 1: This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined. (<button>
                <a href=./boi...</button>)
[DEVENorg (docs/index.html)] error 1: Anchor element found with a valid href attribute, but no link content has been supplied. (<a href=./boiler>
                    <svg width...</a>)
[DEVENorg (docs/index.html)] error 1: Duplicate id attribute value slidePrev found on the web page. (<svg width=18 height=18 viewBox=0 0 36 26 fill=none xmlns=http://www.w3.org/2000/svg id=slidePrev>
                    <path d=M...</svg>)
[DEVENorg (docs/index.html)] error 1: Duplicate id attribute value slideNext found on the web page. (<svg width=18 height=18 viewBox=0 0 36 26 fill=none xmlns=http://www.w3.org/2000/svg id=slideNext>
                    <path d=M...</svg>)
```

</details>
</summary>

**Comment by remmycat** [comment]
As #131 was merged and the leftover task has its own ticket #132, I'm closing this as completed
